### PR TITLE
Update v8 data

### LINKED ISSUE #221: Update Node 8 schedule?
For example, the Node 8 start date in schedule.json still says 2017-04-30

### PR DESCRIPTION
**Pull request by beardedtim**
This PR will resolve #221 by updating `schedule.json` with the information found [here](https://nodejs.org/en/blog/release/v8.0.0/) on the node blog.

### CODE REVIEW

**Comment by hbetts** [comment]
This merge request should close out #221 by @cjihrig

File: schedule.json [hunk 1]
```diff
@@ -32,11 +32,11 @@
     "end": "2017-06-30"
   },
   "v8": {
-    "start": "2017-04-30",
+    "start": "2017-05-30",
     "lts": "2017-10-31",
-    "maintenance": "2019-04-01",
+    "maintenance": "2019-05-01",
```
Comment (hbetts) [inline schedule.json]: Is the maintenance start time delayed? I wasn't able to find any reference to that effect within the blog post.

Comment (beardedtim) [inline schedule.json]: I thought I saw that somewhere but I can't find anything saying that so I am going to change it back.

**Comment by gibfahn** [comment]
Could you add the codename to https://github.com/nodejs/LTS#lts-schedule1 as well?

**Comment by beardedtim** [comment]
@gibfahn I have made the changes to `README`. I could not find a link as direct as the others to a download so I did not add it as a link. Do you want me to link to the `node` homepage?

**Comment by gibfahn** [comment]
That looks fine, I don't think the `latest-carbon` link has been set up yet, maybe it doesn't get done till Node 8 goes LTS.

**Comment by beardedtim** [comment]
I know this is very low priority for this group but any critique or changed that need to be made for this to be mergable?

**Comment by gibfahn** [comment]
@beardedtim thanks for the reminder!

**Pull request merged**